src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    protected String start;
+    protected String end;
+    public Event(String description, String start, String end) {
```

Comment:
Can add an empty line in between variable declarations and function for better visibility.